src/Microsoft.FeatureManagement/Group.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using System.Collections.Generic;
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// The definition of a group allocation.
+    /// </summary>
+    public class Group
```

Comment:
Not sure about the name of the folder since I can't put the Allocation.cs file in an Allocation namespace, so I was gonna try VariantAllocation as the name for now, open to something else.

Reply:
That works for me. If we make it broader, we could do `Variants`. But variants are becoming an important part of things so that might be too big of a folder.

Reply:
@amerjusupovic 

Folder and namespace are unrelated.

Reply:
To be clear, I like Ross's idea to have allocation folder. I just don't think the namespace is needed.

Reply:
My fault I misread that, will add that back in without namespace.